cylc/flow/taskdef.py

```diff
@@ -216,8 +216,6 @@ def get_parent_points(self, point):
             if seq in self.dependencies:
                 # task has prereqs in this sequence
                 for dep in self.dependencies[seq]:
-                    if dep.suicide:
-                        continue
```

Comment:
This was preventing the upstream parent of a suicide trigger being recognized as a parent (matters for spawning if the suicided task has no other parents).